test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -1047,3 +1047,40 @@ func.func @torch.aten.mse_loss$sum_reduction(%arg0: !torch.vtensor<[?,?],f32>, %
   %0 = torch.aten.mse_loss %arg0, %arg1, %int2 : !torch.vtensor<[?,?],f32>, !torch.vtensor<[?,?],f32>, !torch.int -> !torch.vtensor<[?,?],f32>
   return %0 : !torch.vtensor<[?,?],f32>
 }
+
+// -----
+
+// CHECK-LABEL:   func.func @torch.aten.randint.low() -> !torch.vtensor<[1024,1024],si64> {
+// CHECK:         %[[NONE:.*]] = torch.constant.none
+// CHECK:         %[[INT4:.*]] = torch.constant.int 4
+// CHECK:         %[[INT1:.*]] = torch.constant.int 1
+// CHECK:         %[[INT1000:.*]] = torch.constant.int 1000
+// CHECK:         %[[INT1024:.*]] = torch.constant.int 1024
+// CHECK:         %[[SIZE:.*]] = torch.prim.ListConstruct %[[INT1024]], %[[INT1024]] : (!torch.int, !torch.int) -> !torch.list<int>
+// CHECK:         %[[NONE_0:.*]] = torch.constant.none
+// CHECK:         %[[FALSE:.*]] = torch.constant.bool false
+// CHECK:         %[[FLOAT0:.*]] = torch.constant.float 0.000000e+00
+// CHECK:         %[[FLOAT1:.*]] = torch.constant.float 1.000000e+00
+// CHECK:         %[[FLOAT1000:.*]] = torch.constant.float 1.000000e+03
+// CHECK:         %[[VAL:.*]] = torch.prim.NumToTensor.Scalar %[[FLOAT0]] : !torch.float -> !torch.vtensor<[],f64>
+// CHECK:         %[[INT4_1:.*]] = torch.constant.int 4
+// CHECK:         %[[FALSE_2:.*]] = torch.constant.bool false
+// CHECK:         %[[NONE_3:.*]] = torch.constant.none
+// CHECK:         %[[VAL_0:.*]] = torch.aten.to.dtype %[[VAL]], %[[INT4_1]], %[[FALSE_2]], %[[FALSE_2]], %[[NONE_3]] : !torch.vtensor<[],f64>, !torch.int, !torch.bool, !torch.bool, !torch.none -> !torch.vtensor<[],si64>
+// CHECK:         %[[VAL_1:.*]] = torch.aten.broadcast_to %[[VAL_0]], %[[SIZE]] : !torch.vtensor<[],si64>, !torch.list<int> -> !torch.vtensor<[1024,1024],si64>
+// CHECK:         %[[INT7:.*]] = torch.constant.int 7
+// CHECK:         %[[VAL_2:.*]] = torch.aten.to.dtype %[[VAL_1]], %[[INT7]], %[[FALSE]], %[[FALSE]], %[[NONE_0]] : !torch.vtensor<[1024,1024],si64>, !torch.int, !torch.bool, !torch.bool, !torch.none -> !torch.vtensor<[1024,1024],f64>
+// CHECK:         %[[UNIFORM:.*]] = torch.aten.uniform %[[VAL_2]], %[[FLOAT1]], %[[FLOAT1000]], %[[NONE_0]] : !torch.vtensor<[1024,1024],f64>, !torch.float, !torch.float, !torch.none -> !torch.vtensor<[1024,1024],f64>
+// CHECK:         %[[INT4_4:.*]] = torch.constant.int 4
+// CHECK:         %[[RES:.*]] = torch.aten.to.dtype %[[UNIFORM]], %[[INT4_4]], %[[FALSE]], %[[FALSE]], %[[NONE_0]] : !torch.vtensor<[1024,1024],f64>, !torch.int, !torch.bool, !torch.bool, !torch.none -> !torch.vtensor<[1024,1024],si64>
+// CHECK:         return %[[RES]] : !torch.vtensor<[1024,1024],si64>
+func.func @torch.aten.randint.low() -> !torch.vtensor<[1024,1024],si64> {
```

Comment:
No need for this test. This falls under:

> DON'T use a unit test if your lowering pattern could be described as a trivial "macro expansion" of one op into another op or set of ops.

https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing